src/pybel/io/umbrella_nodelink.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+"""Conversion functions for customized node-link JSON format prior canonicalization.
```

Comment:
Again, this one liner really doesn't help the user to understand what this schema is. We can't merge this code into master when it will make new users more confused.

Also, the supporting text under it still doesn't spend any time taking about the rationale for why this type of export exists. This is the place to write it like academic text (imagine I will copy/paste it into the PyBEL 2020 Update paper. What would a user need to know about why this function exists, and what problem it helps solve? Why are previous exporters not good enough/what are they lacking?)

Reply:
I think it should be clear now